util/cron/common.bash

```diff
@@ -101,8 +101,12 @@ export CHPL_NIGHTLY_CRON_LOGDIR=$CHPL_NIGHTLY_LOGDIR
 
 # It is tempting to use hostname --short, but macs only support the short form
 # of the argument.
-export CHPL_TEST_PERF_DIR=$logdir_prefix/NightlyPerformance/$(hostname -s)
-export CHPL_TEST_COMP_PERF_DIR=$logdir_prefix/NightlyPerformance/$(hostname -s)
+if [ -z "$CHPL_TEST_PERF_DIR" ]; then
+    export CHPL_TEST_PERF_DIR=$logdir_prefix/NightlyPerformance/$(hostname -s)
+fi
+if [ -z "$CHPL_TEST_COMP_PERF_DIR"]; then
```

Comment:
I think you need a space between the closing quote and the closing square bracket here. Otherwise, bash will complain about a missing ].